angr/analyses/binary_optimizer.py

```diff
@@ -460,7 +460,7 @@ def _register_reallocation(self, function, data_graph):
         # make sure we never gets the address of those stack variables into any register
         # say, lea edx, [ebp-0x4] is forbidden
         # check all edges in data graph
-        for src, dst, data in data_graph.edges(data=True):
+        for src, dst, _data in data_graph.edges(data=True):
```

Comment:
If `data` is not used, we should change it to `for src, dst in data_graphs.edges():`.

Reply:
Thanks